libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -273,6 +273,8 @@ const AP_GPS_UBLOX::config_list AP_GPS_UBLOX::config_L5_ovrd_dis[] {
     {ConfigKey::CFG_SIGNAL_L5_HEALTH_OVRD, 0},
 };
 
+AP_GPS_UBLOX::config_list AP_GPS_UBLOX::config_GNSS[UBLOX_MAX_GNSS_CONFIG_BLOCKS*3] {};
```

Comment:
certainly should not be initialised, but would be much better to be dynamically allocated as needed, and freed when done, step skipped if we can't allocate the memory

Reply:
Fixed